src/Service/Extensions/ConversionFunctions.cs

```diff
@@ -0,0 +1,9 @@
+namespace Passwordless.Service.Extensions;
+
+public static class ConversionFunctions
+{
+    public static TimeSpan? GetNullableTimeSpanFromSeconds(this int? seconds) =>
```

Comment:
Nit: if you change the signature to `public static TimeSpan? ToTimeSpanFromSeconds(this int seconds)` (non nullable), then you can update the callsite like this:

```csharp
var timespan = timeToLive?.ToTimeSpanFromSeconds();
```

Which is subjectively a bit cleaner because it doesn't look like you're calling a method on a potentially null reference.